### DISCUSSION TITLE
does model check support MF6?

### DISCUSSION THREAD

**Discussion by StellaLiu-gogogo** [modflowpy/flopy#1754] [Q&A] [answered]
Hi there. 
I'm using flopy to create a gwm in modflow6. 
after generating the modflow input file it fails to run due to an floating overflow. 

Thus I'd like to perform a model check. 
I tried to follow the iphython notebook exmple to first load my model as 
```python
m = fp.modflow.Modflow.load('{mymodelname}.nam')
m.check()
```
However, it returns ValueError: line number 2 has fewer than 3 items: BEGIN options

I tried to also appy the m.check with a tutorial model file. it returns same error. 
So I suppose it is caused by the difference nam file format of mf6 with the previous modflow version?

Thus I came here to confirm if the .check() supports mf6?
Or is there any other way to perform a model check for modflow 6?

Cheers
Stella

**Comment by spaulins-usgs** [comment, marked as answer]
Flopy supports running check on a MF6 simulation.  It looks like you are using an example that loads a MF2005 model.  Loading and checking an MF6 simulation looks something like this:

    sim = flopy.mf6.MFSimulation.load(sim_ws=ws)
    sim.check()

Where sim_ws is the path to the folder that contains the simulation nam file.

Note that the example notebooks that are specific to MF6 start with "flopy3_mf6".   While some of the other notebooks include MF6 examples (plotting), most of the MF6 example code can be found in the "flopy3_mf6" notebooks.  Offhand I am not sure if there is a notebook with an example of check for MF6, but the code above should work.

**Reply by StellaLiu-gogogo** [reply]
aha. I see. 
Thanks!
